agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -1456,3 +1456,48 @@ func TestServerEndpointValidator(t *testing.T) {
 	assert.True(t, ok, "Get exit code failed")
 	assert.Equal(t, 42, code, "Wrong exit code")
 }
+
+// TestAppMeshCNIPlugin validates the functionality of appmesh cni plugin.
+func TestAppMeshCNIPlugin(t *testing.T) {
+	cwlClient := cloudwatchlogs.New(session.New(), aws.NewConfig().WithRegion(*ECS.Config.Region))
+	// Test whether the log group exists or not
+	respDescribeLogGroups, err := cwlClient.DescribeLogGroups(&cloudwatchlogs.DescribeLogGroupsInput{
+		LogGroupNamePrefix: aws.String(awslogsLogGroupName),
+	})
+	require.NoError(t, err, "CloudWatchLogs describe log groups failed")
+	logGroupExists := false
+	for i := 0; i < len(respDescribeLogGroups.LogGroups); i++ {
+		if *respDescribeLogGroups.LogGroups[i].LogGroupName == awslogsLogGroupName {
+			logGroupExists = true
+			break
+		}
+	}
+
+	if !logGroupExists {
+		_, err := cwlClient.CreateLogGroup(&cloudwatchlogs.CreateLogGroupInput{
+			LogGroupName: aws.String(awslogsLogGroupName),
+		})
+		require.NoError(t, err, fmt.Sprintf("Failed to create log group %s", awslogsLogGroupName))
+	}
+
+	agent := RunAgent(t, &AgentOptions{
+		EnableTaskENI: true,
+		ExtraEnvironment: map[string]string{
+			"ECS_AVAILABLE_LOGGING_DRIVERS": `["awslogs"]`,
+		},
+	})
+	defer agent.Cleanup()
+	agent.RequireVersion(">=1.25.3")
```

Comment:
Is this feature added in 1.25.3? if not, add a TODO to update the  version.

Reply:
In 1.26.0, I will add a todo there.